code/modules/awaymissions/maploader/reader.dm

```diff
@@ -358,28 +358,30 @@ GLOBAL_DATUM_INIT(_preloader, /datum/dmm_suite/preloader, new())
 /datum/dmm_suite/proc/readlist(text, delimiter = ",")
 	var/list/to_return = list()
 
-	var/position
+	var/delimiter_position
 	var/old_position = 1
 
 	do
 		// find next delimiter that is not within  "..."
-		position = find_next_delimiter_position(text, old_position, delimiter)
+		delimiter_position = find_next_delimiter_position(text, old_position, delimiter)
 
 		// check if this is a simple variable (as in list(var1, var2)) or an associative one (as in list(var1="foo", var2=7))
-		var/equal_position = findtext(text, "=", old_position, position)
+		var/equal_position = findtext(text, "=", old_position, delimiter_position)
 
-		var/trim_left = trim_text(copytext(text, old_position, (equal_position ? equal_position : position)), 1) // the name of the variable, must trim quotes to build a BYOND compliant associatives list
-		old_position = position + 1
+		// Take the left value of the association or just the value if it isn't an association
+		var/left_value = copytext(text, old_position, (equal_position ? equal_position : delimiter_position))
```

Comment:
Here it doesn't use `trim_text(..., TRUE)` anymore. So it doesn't trim the quotes